onnxruntime/core/providers/cpu/cpu_execution_provider.cc

```diff
@@ -347,7 +347,7 @@ class ONNX_OPERATOR_KERNEL_CLASS_NAME(kCpuExecutionProvider, kOnnxDomain, 11, Un
 class ONNX_OPERATOR_KERNEL_CLASS_NAME(kCpuExecutionProvider, kOnnxDomain, 11, Det);
 class ONNX_OPERATOR_KERNEL_CLASS_NAME(kCpuExecutionProvider, kOnnxDomain, 11, ScatterElements);
 class ONNX_OPERATOR_KERNEL_CLASS_NAME(kCpuExecutionProvider, kOnnxDomain, 11, NonMaxSuppression);
-
+class ONNX_OPERATOR_KERNEL_CLASS_NAME(kCpuExecutionProvider, kOnnxDomain, 11, Gemm);
 
```

Comment:
There's some overlap with [https://github.com/microsoft/onnxruntime/pull/1956](https://github.com/microsoft/onnxruntime/pull/1956) and may need an update to the opset 9 declaration if that gets merged first.

Reply:
Fixed.